signer/types.go

```diff
@@ -64,9 +64,17 @@ func (k KeySpec) SignatureAlgorithm() SignatureAlgorithm {
 	return ""
 }
 
+// SigningScheme formalizes the feature set (guarantees) provided by the signature.
+type SigningScheme string
+
+const (
+	SigningSchemeX509Default          SigningScheme = "notary.default.x509"
+	SigningSchemeX509SigningAuthority SigningScheme = "notary.signingAuthority.x509"
```

Comment:
These were changed to `notary.x509` and `notary.x509.signingAuthority` in the latest spec. See https://github.com/notaryproject/notaryproject/blob/main/signature-specification.md#standard-attributes

Reply:
Yep, waiting for @gokarnm to review and if there are any feedback will squash this renaming with other changes.